deep_quoridor/src/renderers/computation_times.py

```diff
@@ -31,6 +31,8 @@ def end_arena(self, game, results: list[GameResult]):
                     f"{max(computation_times_ms):.3f}",
                     f"{statistics.median(computation_times_ms):.3f}",
                 ]
+                if num_moves > 0
```

Comment:
Thanks for fixing this! I didn't encounter the problem often since it usually crashed earlier :-)

Reply:
:handshake: